index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio ErnestKorir</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700;800&display=swap" rel="stylesheet">
+</head>
+
+<body>
```

Comment:
- [ ] Please note that the minimum mobile screen size should be `375`px , kindly make sure that your website is well centered and aligned in the width of 375px+

![fixxxxxxxxxxxxxxxxxxxxxxxx](https://user-images.githubusercontent.com/89282221/192089004-d8da6ed8-e828-4436-b8d3-e3a501ef843a.gif)